bindings/node/native/src/classes/account_manager/mod.rs

```diff
@@ -179,9 +191,9 @@ declare_types! {
             };
             match account {
                 Ok(acc) => {
```

Comment:
hard to find anything to criticize, especially when not too familiar with the code base, but my preference here would be to just use the same name (`account`) on the inner variable again. Specially this abbreviation is also used to abbreviate `accumulator`.